coalib/settings/ConfigurationGathering.py

```diff
@@ -53,25 +53,27 @@ def load_config_file(filename, log_printer, silent=False):
                         ``False``.
     """
     filename = os.path.abspath(filename)
-
+    strict = True  # TODO: Remove it (Just for testing purposes)
     try:
         return ConfParser().parse(filename)
     except FileNotFoundError:
         if not silent:
-            if os.path.basename(filename) == Constants.default_coafile:
+            if (not strict) and \
```

Comment:
This wasn't the point of adding a "strict mode". It was to enable some warnings like glob pattern not matching any file as errors. I don't see any warning -> error changes. Please discuss in gitter if you need further insight before submitting a PR.